app/scripts/views/choose_what_to_sync.js

```diff
@@ -121,6 +123,8 @@ define(function (require, exports, module) {
         });
       }
     }
+  }, {
+    SCREEN_CLASS
```

Comment:
I'm curious, what's the reason for including `SCREEN_CLASS` as a property here? I only see it referenced directly in this file...